src/script/squirrel_helper.hpp

```diff
@@ -16,7 +16,7 @@
 #include "../core/convertible_through_base.hpp"
 #include "squirrel_helper_type.hpp"
 
-template <class CL, ScriptType ST> const char *GetClassName();
+template <class CL, ScriptType ST> std::optional<std::string> GetClassName();
```

Comment:
Does it need to be `std::string` (does the caller need to own the string) or will `std::string_view` suffice?